apis/v1alpha1/iam_request.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+// IAMRequest represents a request to update IAM policies.
+type IAMRequest struct {
+	// List of ResourcePolicy, each specifies the IAM principals/members to role
+	// bindings to be added for a GCP resource IAM policy.
+	ResourcePolicies []*ResourcePolicy `json:"policies,omitempty"`
```

Comment:
Forgot to mention, do we need `yaml` tag?

Reply:
Good catch, will change to yaml tag instead!